packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -94,6 +94,11 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<PersonaBase |
    */
   presence?: PersonaPresence;
 
+  /**
+   * Presence title of the person to be shown as a tooltip on hover over the presence icon.
```

Comment:
```suggestion
   * Presence title to be shown as a tooltip on hover over the presence icon.
```
I think the "of the person" part makes it somewhat confusing.